Controllers/BookmarksController.cs

```diff
@@ -0,0 +1,95 @@
+using bootcamp_api.Models;
+using bootcamp_api.Services;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
+
+namespace bootcamp_api.Controllers;
+
+[ApiController]
+[Produces("application/json")]
+[Consumes("application/json")]
+[SwaggerResponse(500, "An error occurred while processing the request", typeof(ProblemDetails))]
```

Comment:
You shouldn't need a controller level swagger response. We don't need to tell the user we expect 500 errors.

Reply:
I removed these swagger responses in the latest commit.